### DISCUSSION TITLE
[Feature] Folders and Tags in Partner Sharing

### DISCUSSION THREAD

**Discussion by wonx** [immich-app/immich#12374] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Now that we have Tags and Folders available in Immich, it would be interesting to be able to see those folders and tags for libraries that were shared by a partner. The same way we can choose to show their library in our timeline, there could be another toggle to show Tags and Folders in ours.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#6350
